pkg/apis/autoscaling/v1alpha1/metric_types.go

```diff
@@ -63,7 +64,9 @@ type MetricSpec struct {
 }
 
 // MetricStatus reflects the status of metric collection for this specific entity.
-type MetricStatus struct{}
+type MetricStatus struct {
```

Comment:
FWIW, we can just do `type MetricStatus duckv1beta1.Status`?

Reply:
@vagababov I'm not sure if type alias is appropriate here, since it's for gradual code repair while moving a type between packages during large-scale refactoring